lisa/executable.py

```diff
@@ -278,6 +278,32 @@ def run(
             expected_exit_code_failure_message=expected_exit_code_failure_message,
         )
 
+    def run_script_async(
```

Comment:
What's the reason to add it in tools? Follow the `withscript.py` to use the `CustomScriptBuilder` to define, and use `run_async` of `CustomScript`.

Reply:
The reason I added in it tools is to avoid setup instruction in the script. If we call the script from tool, it will take care of installation and then call the script

Reply:
The tool can be called itself, I don't get why it needs to be called from the tool. It looks the script depends on the tool here, so add dependency in the `CustomScriptBuilder`, so when it's used the tool will be installed.

Reply:
This makes sense and should be cleaner. I will update!